sdk/storage/storage-blob/test/node/blobclient.spec.ts

```diff
@@ -686,3 +689,200 @@ describe("BlobClient Node.js only", () => {
     });
   });
 });
+
+describe("BlobClient Node.js Only - ImmutabilityPolicy", () => {
+  let blobServiceClient: BlobServiceClient;
+  let containerName: string;
+  let containerClient: ContainerClient;
+  let blobName: string;
+  let blobClient: BlobClient;
+  const content = "Hello World";
+
+  let recorder: Recorder;
+
+  beforeEach(async function() {
+    recorder = record(this, recorderEnvSetup);
+    blobServiceClient = getBSU();
+    try {
+      containerName = getImmutableContainerName();
+    } catch {
+      this.skip();
+    }
```

Comment:
create an issue to track live test enabling for this feature

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/16208